lib/hlclock/timestamp.ex

```diff
@@ -97,6 +130,19 @@ defmodule HLClock.Timestamp do
     %T{time: t, counter: c, node_id: n}
   end
 
+  @doc """
+  Recover a Timestamp from the string representation.
```

Comment:
This is really dumb but can we provide an example here so its clear what the output of this should look like?